tasks/am-devtools.yml

```diff
@@ -12,6 +12,45 @@
     - "graphviz"
     - "libgraphviz-dev"
     - "python-pygraphviz"
+  when:
+    - ansible_distribution == "Ubuntu"
+
+- name: "Install am-devtools package dependencies (RH/CentOS)"
+  yum:
+    pkg: "{{ item }}"
+    state: "latest"
+  with_items:
+    - "rubygems"
+    - "ruby-devel"
+    - "graphviz"
+    - "graphviz-devel"
+    - "graphviz-python"
+    - "python2-pkgconfig"
+  when:
+    - ansible_distribution == "RedHat" or ansible_distribution == "CentOS"
+
+# Needed to install ronn on RH/CentOS
+- name: "Install psych 2.2.4 (RH/CentOS)"
+  command: "gem install psych -v 2.2.4"
```

Comment:
The package version should be in a variable, so it's easier to change

Reply:
There's also the `gem` module, e.g.:

```
- gem:
  name: psych
  state: latest
```